kahypar/partition/partitioner.h

```diff
@@ -187,6 +190,10 @@ inline void Partitioner::setupContext(const Hypergraph& hypergraph, Context& con
                                                     * hypergraph.totalWeight());
   context.setupPartWeights(hypergraph.totalWeight());
 
+  context.partition.max_he_size_threshold =
+    std::max(hypergraph.initialNumNodes() *
+      context.partition.max_he_size_threshold_factor, 50000.0);
```

Comment:
Can we expose this `50000.0` as a dedicated tuning parameter as well?

Reply:
ungern, but done